content/swiftui/concepts/Lists.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: "SwiftUI List" 
```

Comment:
just 'List' is good here. Also, make sure to lowercase the file name and place it within a `lists` folder.